NuKeeper.Inspection.Tests/RepositoryInspection/ProjectFileReaderTests.cs

```diff
@@ -86,6 +87,21 @@ public void EmptyProjectCanBeRead()
             Assert.That(packages, Is.Empty);
         }
 
+        [Test]
+        public void UnusualEncodingProjectCanBeRead()
+        {
+            const string NoProject =
+                @"<?xml version=""1.0"" encoding=""Windows-1252""?>
```

Comment:
Is this a thing that you came across?

Reply:
Comes from user issue #365 - but I should have used the VS classic project in the repro here, not the core SDK one.

Reply:
Checked with a dotnet core project - technically this is a valid encoding and it works with dotnetcore sdk, so I'd keep this. Test case is valid, even if unusual.